fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -163,4 +170,27 @@ private File fileForContainer(final URI uri) {
         return new File(config.getDescriptionDirectory(), uri.getPath().replaceAll(":", "_")
             + config.getRdfExtension());
     }
+
+    /**
+     * Checks the response code and throws a RuntimeException with a helpful
+     * message (when possible) for non 2xx codes.
+     * @param response the response from a REST call to Fedora
+     * @param uri the URI against which the request was made
+     */
+    private void checkValidResponse(final FcrepoResponse response, final URI uri) {
+        if (response.getStatusCode() >= 200 && response.getStatusCode() < 300) {
```

Comment:
I'm just going on and on with this stuff, but I would rather read a `switch` here:
```
switch (response.getStatusCode()) {
    case 401:
        throw new AuthenticationRequiredRuntimeException();
    case 403:
        throw new RuntimeException("Export operation failed: authorization denied for " + config.getUsername());
    case 404:
        throw new RuntimeException("Export operation failed: resource not found " + uri);
    default:
         if (response.getStatusCode() < 200 || response.getStatusCode() => 300) {
             throw new RuntimeException("Export operation failed: unexpected status " + response.getStatusCode() + " for " + uri);
         }
}
```